### ISSUE TITLE
FR: Move live preview beneath header

### ISSUE THREAD

**Issue by Mathias-Syversen** [craftcms/cms#5547]
The new blurred header in Craft 3.4 is really nice, but in live preview it covers the top 54px of the iframe.
This results in page navigation being covered up:

![Screenshot 2020-01-30 at 13 47 52](https://user-images.githubusercontent.com/53942798/73451389-cc823800-4367-11ea-8e86-7dcd833f5df9.png)

Something like this would be preferable:

![Screenshot 2020-01-30 at 13 48 36](https://user-images.githubusercontent.com/53942798/73451422-dc9a1780-4367-11ea-9c5b-e246ec00586a.png)

**Comment by brandonkelly** [comment]
Yeah I was a bit on the fence on whether it was a good idea. We could try shrinking the size of the iframe a little and then increasing it to full height as you scroll down, however that would only work when the preview URL is on the same domain as the control panel.

Alternatively you could add some top padding to your webpage if it’s a live preview request:

```twig
{% set padded = craft.app.request.getQueryParam('x-craft-live-preview') != null %}
<body {% if padded %}class="padded"{% endif %}>
```

**Comment by wsydney76** [comment]
Supporting the idea that the page content should never be hidden behind a header.

* The content being edited may affect the header, especially its responsive behavior
* It can look really weird

![image](https://user-images.githubusercontent.com/60010597/73841510-1f059d80-481b-11ea-8db0-8d6706e10d48.png)

> Alternatively you could add some top padding...

Our devs say: In addition you have to figure out if there is a header at all, taking dynamically added targets into account. Out of scope escpecially for headless preview.

/Mel 

PS.
This only happens in Chrome, in Firefox 72.0.2 it works great:

![image](https://user-images.githubusercontent.com/60010597/73841905-dac6cd00-481b-11ea-8a26-1641ca9e6d42.png)

**Comment by dennisfrank** [comment]
I am also not so happy with the Live Preview Target selector at the top of the preview frame. So I moved it with some custom CSS to the left: 

```css
.lp-preview-container header {
  background: none;
  -webkit-backdrop-filter: none;
          backdrop-filter: none;
  box-shadow: none;
}

.lp-preview-container header .btn.menubtn {
  position: absolute;
  top: 10px;
  right: calc(100% + 10px);
}
```

![image](https://user-images.githubusercontent.com/65968/74495530-5b4e9300-4ed8-11ea-87ad-6496a0a56d50.png)

**Comment by brandonkelly** [comment]
@dennisfrank can you please post about that in a separate issue?

**Comment by brandonkelly** [comment]
This is now resolved for Craft 3.5. The header no longer covers the preview frame when it’s scrolled all the way to the top, but still maintains its translucent blur effect as you scroll down the frame. As a bonus, maintaining the scroll position across refreshes is now much more reliable, as the iframe is no longer scrolled directly; only its container div.

(Using the neat [iFrame Resizer](https://github.com/davidjbradshaw/iframe-resizer) library to pull it off.)
